scripts/calc_performance_metrics.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/python3
+
+import sys, traceback
+
+if len(sys.argv) <= 1:
+    print("Please specify the run_output file")
+    exit(-1)
+
+seq_len = 128
+if len(sys.argv) > 2:
+    seq_len = int(sys.argv[2])
+batch_size = 1
+if len(sys.argv) > 3:
+    batch_size = int(sys.argv[3])
+ngpus = 1
+if len(sys.argv) > 4:
+    ngpus = int(sys.argv[4])
+
+try:
+    with open(sys.argv[1]) as f:
+        sum = 0
+        count = 0
+        for l in f.readlines():
+            if not "global_step/sec" in l:
```

Comment:
people usually use regex to match lines

Reply:
But it is fine if it works

Reply:
Never liked Regex. This is more pythonian. :-) I will fix the typo and move it in.